ota-plus-web/app/reactapp-new/src/store/feed/types.ts

```diff
@@ -0,0 +1,18 @@
+export interface FeedResource {
+  name?: string;
+  groupName: string;
+}
+export interface FeedData {
+  createdAt: string;
+  resource: FeedResource;
+  _type: string;
```

Comment:
is _type private? If no then why underscore convention?

Reply:
this is how it comes from API